### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#3994] [Releases & Announcements]
## Changelog

-  Automations ( Beta )
-  Custom Views for conversations page
-  Custom Filters for contacts page
-  Email Signatures
-  Support Bandwidth as a provider for SMS channel
-  Ability to configure active storage direct uploads
-  Ability to send multiple attachments from the agent dashboard
-  Super Admin dashboard enhancements 
-  Support for interactive messages in Whatsapp channel ( 360 Dialog )
-  Ability to group report data by weekly, monthly and yearly filters
-  Ability to toggle conversation continuity via email for website channel
-  Resolve action for Dialogflow integration
-  SSL Verify mode and email encryption settings for SMTP mailboxes
-  Vue-router for widget route management
-  Numerous bug fixes and enhancements

Thanks to @ximenasandoval, @shivamsinghchahar, @iprithvitharun, @freddy36, @sagararyal, @hostup, @robguthrie,   for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.2.0'>v2.2.0</a>.</em>

**Comment by johanndrews** [comment]
Hi. Maybe I overlook somthing. I updated to 2.2.0, but don't see any mail signature options within my profile?

**Reply by sojan-official** [reply]
@johanndrews  please see if this is helpful

![Kapture 2022-02-21 at 18 17 44](https://user-images.githubusercontent.com/73185/154958464-5a1647a1-1a3a-42f7-8c6e-b1247a95ae53.gif)

**Reply by johanndrews** [reply]
@sojan-official unfortunately I don‘t have the shown Area in my profile. Should I try to compile again? Or what could be gone wrong, that my frontend don‘t show the last update?

**Reply by sojan-official** [reply]
@johanndrews most likely. Could you retry compiling the assets .

**Reply by johanndrews** [reply]
![7BB6A63D-12BF-4F74-BAE6-D28AB3237D16](https://user-images.githubusercontent.com/37542352/154985157-ceef29bc-09ac-414d-b752-fec80ad7ad4f.png)
Compiling failed. I had previously only installed 2.1 on a blank Ubunto 20.04 System. I use the nginx proxy, as described, with certbot. Nothing else.
Already tried to reinstall. But on an complete new instance, using the setup.sh script I get an compiling failure, too.

**Reply by sojan-official** [reply]
@johanndrews  could you provide the spec of the ubuntu machine that you are using ?  mainly the memory limit.  
also is there any specific errors that show up around `compilation_failed` in the screen shot i can see only the warnings

**Reply by johanndrews** [reply]
The line before are only the INFO posts, which files are compiling. All that comes after compiling failed, is what you see in the screenshot.
1 vCPU, 2 GB Ram, an Hetzner Cloud CX11 Server

**Reply by sojan-official** [reply]
@johanndrews could be a memory issue, we usually recommend 4gb instances. But if you don't expect too much traffic on the instance. You could get through the compiling step by increasing the swap memory. 

ref: https://github.com/chatwoot/chatwoot/issues/1700

**Reply by johanndrews** [reply]
This helped! I rescaled to 4GB Ram. Don‘t thought of this, as we don‘t have that much conversations. Thanks a lot!

**Comment by nicolastefanac** [comment]
I think conversation continuity finally works without a s3 storage configured, now im happy!
